quickfixj-spring-boot-starter/src/test/java/io/allune/quickfixj/spring/boot/starter/connection/SessionSettingsLocatorTest.java

```diff
@@ -43,9 +43,8 @@ public void shouldLoadDefaultFromSystemProperty() throws URISyntaxException {
 		SessionSettings settings = sessionSettingsLocator.loadSettings("classpath:quickfixj.cfg", null, null, null);
 		assertThat(settings).isNotNull();
 
-		URL resource = SessionSettingsLocatorTest.class.getResource("/quickfixj.cfg");
-		File file = Paths.get(resource.toURI()).toFile();
-		settings = sessionSettingsLocator.loadSettings(null, null, "file://" + file.getAbsolutePath(), null);
+		File file = new File(SessionSettingsLocatorTest.class.getClass().getResource("/quickfixj.cfg").getFile());
```

Comment:
I think `getClass()` is redundant, and probably why the build is failing locally for me (Linux)

Reply:
Yes, it's redundant, removing.